src/ssl.c

```diff
@@ -5012,6 +5022,12 @@ int wolfSSL_Init(void)
 #endif
 
 #ifdef OPENSSL_EXTRA
+        /* OpenSSL registers cleanup using atexit */
+        if (atexit(AtExitCleanup) != 0) {
```

Comment:
This is not portable and I would recommend against it.

Reply:
Added a configure.ac check for `atexit`. Would that work?